azkaban-common/src/main/java/azkaban/project/NodeBeanLoader.java

```diff
@@ -35,11 +34,16 @@
  */
 public class NodeBeanLoader {
 
-  public NodeBean load(final File flowFile) throws FileNotFoundException {
+  public NodeBean load(final File flowFile) throws Exception {
+    checkArgument(flowFile != null);
```

Comment:
Just another way to handle: checkArgument(flow != null && flowFile.exists());

Reply:
Will consolidate the code, thanks!